src/libstd/keyword_docs.rs

```diff
@@ -925,8 +925,14 @@ mod mut_keyword {}
 //
 /// Make an item visible to others.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
-///
+/// The keyword `pub` makes any module, function, or data structure accessible from inside
+/// of external modules. The `pub` keyword may also be used in a `use` declaration to re-export
+/// an identifier from a namespace.
+///
+/// For more information on the `pub` keyword, please see the [Reference] section
+/// of the Reference. Or the [Rust by Example] section of Rust By Example.
+/// [Reference]: ../reference/visibility-and-privacy.html?highlight=pub#visibility-and-privacy
```

Comment:
```suggestion
/// For more information on the `pub` keyword, please see the visibility section
/// of the [reference] and for some examples, see [Rust by Example].
/// 
/// [reference]: ../reference/visibility-and-privacy.html?highlight=pub#visibility-and-privacy
```

Reply:
Just by reading the error message I don't think I would have figured out that the capitalization in [reference] was the problem. Thanks for this suggestion!